lib/process/processWrapper.js

```diff
@@ -71,9 +74,10 @@ class ProcessWrapper {
 
   send() {
     if (!process.connected) {
-      return;
+      return false;
     }
     process.send(...arguments);
```

Comment:
Better idea still, put the return here, I think process.sen returns a success

Reply:
> subprocess.send() will return false if the channel has closed or when the backlog of unsent messages exceeds a threshold that makes it unwise to send more. Otherwise, the method returns true. The callback function can be used to implement flow control.